docs/ENVIRONMENT.md

```diff
@@ -441,3 +441,10 @@ SYSTEMD_HOME_DEBUG_SUFFIX=foo \
   use for LUKS home directories, overriding the built-in default mount
   options. There's one variable for each of the supported file systems for the
   LUKS home directory backend.
+
+`kernel-install`:
+
+* `$KERNEL_INSTALL_ENABLE` – If set to "0", execution of kernel-install if
```

Comment:
hmm, maybe call this KERNEL_INSTALL_BYPASS?

Reply:
That's the word I was looking for